libs/ui/alert-dialog/helm/src/lib/hlm-alert-dialog-cancel-button.directive.ts

```diff
@@ -1,23 +1,18 @@
-import { Directive, computed, inject, input } from '@angular/core';
-import { HlmButtonDirective } from '@spartan-ng/ui-button-helm';
+import { Directive, computed, input } from '@angular/core';
+import { HlmButtonDirective, provideBrnButtonConfig } from '@spartan-ng/ui-button-helm';
 import { hlm } from '@spartan-ng/ui-core';
 import type { ClassValue } from 'clsx';
 
 @Directive({
 	selector: 'button[hlmAlertDialogCancel]',
 	standalone: true,
 	hostDirectives: [HlmButtonDirective],
+	providers: [provideBrnButtonConfig({ variant: 'outline' })],
 	host: {
-		'[class]': '_computedClass()',
+		'[class]': 'computedClass()',
 	},
 })
 export class HlmAlertDialogCancelButtonDirective {
-	private readonly _hlmBtn = inject(HlmButtonDirective, { host: true });
-
-	public readonly userClass = input<ClassValue>('', { alias: 'class' });
-	protected readonly _computedClass = computed(() => hlm('mt-2 sm:mt-0', this.userClass()));
-
-	constructor() {
-		this._hlmBtn.variant = 'outline';
-	}
+	readonly userClass = input<ClassValue>('', { alias: 'class' });
```

Comment:
Minor here and a non blocker, but we should talk about code style and be consistent in either always be using the public  keyword or not

Reply:
I can add the modifier back in - we can enforce things like this with linting, Im curious, linting seems in a bit of a confusing state, where we have eslint in the project.json files, and biome lint and oxc lint as well. It might be good to only have one linter in place and we can then setup rules to enforce consistency.

I don't mind which linter we use, I guess the only downside with biome lint is it doesn't have any angular template linting and we can't yet write custom rules which could be useful to enforce other conventions like consistent naming for `userClass` or `_computedClass` for example - at least when I last checked that wasn't possible, but that may have since changed. But it would be great to automate this one way or another 😊

Reply:
I started a discussion #458 so we can discuss what we might want to do to sort the linting tool issue